server/routes/recat.js

```diff
@@ -243,6 +245,48 @@ module.exports = function Index({ formService, offendersService, userService, au
 
   const choosingHigherCategory = (current, newCat) => catMap.has(current + newCat)
 
+  router.post(
+    '/review/:bookingId',
+    asyncMiddleware(async (req, res, transactionalDbClient) => {
+      const { bookingId } = req.params
+      const section = 'recat'
+      const form = 'review'
+      const formPageConfig = formConfig[section][form]
+
+      const formData = await formService.getCategorisationRecord(bookingId, transactionalDbClient)
+
+      const suggestedCategory = R.path(['formObject', 'recat', 'decision', 'category'], formData)
+      if (suggestedCategory) {
+        if (suggestedCategory !== 'D' && suggestedCategory !== 'J') {
+          log.info(`Categoriser creating recat categorisation record:`)
+          await formService.setAwaitingApproval(bookingId, transactionalDbClient)
+
+          const nextReviewDate = R.path(['formObject', 'recat', 'nextReviewDate', 'date'], formData)
+          await offendersService.createInitialCategorisation({
+            token: res.locals.user.token,
+            bookingId,
+            suggestedCategory,
+            overriddenCategoryText: 'Cat-tool Recat',
+            nextReviewDate,
+          })
+
+          const nextPath = getPathFor({ data: req.body, config: formPageConfig })
+          res.redirect(`${nextPath}${bookingId}`)
+        } else {
```

Comment:
Actually im not sure about this - maybe the open conditions flag should be set by the decision page